Clients.md

```diff
@@ -0,0 +1,74 @@
+# Featured Client Requirements
+Authored by: ShowierData9978 <contact@showierdata.xyz> on 4/18/24 (CT).
+
+## Terms/Definitions
+
+- Us/We - The Meower team
+
+## Rules
+- Clients created for malicious intent will not be featured
+- Clients must have their own Privacy Policy
```

Comment:
Why? If they don't talk to anything but Meower, there's no need for an extra privacy policy.